app/models/spotlight/search.rb

```diff
@@ -52,22 +50,30 @@ def images
   end
 
   def as_json(*args)
-    super.merge(featured_image: featured_image, count: count)
-  end
-
-  def default_featured_item_id
-    images.first.first if images.present?
+    super.merge(count: count)
```

Comment:
:+1: makes sense to me, and I can't imagine where else we'd serialize these to JSON..